content/_data/changelogs/weekly.yml

```
@@ -5223,6 +5223,74 @@
   # pull: 4303 too minor (PR title: Use actual logger to record Jenkins home and where it was found)
   # pull: 4313 (PR title: [JENKINS-58625] Cancel older builds of a given branch)
 
+- version: '2.203'
+  date: 2019-11-05
+  changes:
+
+# Major RFE
+  - type: major rfe
+    issue: 19887
+    message: |-
+      Allow time zone to be set on a per user basis
+    pull: 4113
+    authors:
+      - silent-snowman
+
+  # RFEs
+
+  - type: rfe
+    message: |-
+      Minor improvements to the logging UI
+    pull: 4305
+    authors:
+      - daniel-beck
+  - type: rfe
+    message: |-
+      Update the Plugin Manager Updates View with more Info about incompatible dependencies and change CSS classes
+    pull: 4299
+    authors:
+      - andipabst
+  - type: rfe
+    issue: 17459
+    message: |-
+      Clicking red/yellow/green icons link to the build Console
+    pull: 4312
+    authors:
+      - rcsalome
+
+  # Bug fixes
+
+  - type: bug
+    issue: 59775
+    message: |-
+      Ensure that <code>PluginManager#start()</code> runs as SYSTEM
+    pull: 4320
+    authors:
+      - Dohbedoh
+  - type: bug
+    issue: 59859
+    message: |-
+      Make resource root URL tokens work with user names containing ":"
+    pull: 4307
+    authors:
+      - daniel-beck
+  - type: bug
+    issue: 59904
+    message: |-
+      Avoid <code>NullPointerException</code> when clearing stale cookies and when there are no cookies in the request
+    pull: 4315
+    authors:
+      - mawinter69
+
+  # Developer-facing changes (APIs, extensions, etc.)
+
+  - type: rfe
+    message: |-
+      Make some <code>ResourceDomainConfiguration</code> API public
```

It has always been public, just restricted.

```suggestion
      Developer: Make some methods in <code>ResourceDomainConfiguration</code> accessible from plugins.
```